tests/phpunit/unit-tests/admin/tools/class-llms-test-admin-tool-recurring-payment-rescheduler.php

```diff
@@ -0,0 +1,254 @@
+
+		$res = LLMS_Unit_Test_Util::call_method( $this->main, 'get_description' );
+		$this->assertTrue( ! empty( $res ) );
+		$this->assertTrue( is_string( $res ) );
+
+	}
+
+	/**
+	 * Test get_label()
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function test_get_label() {
+
+		$res = LLMS_Unit_Test_Util::call_method( $this->main, 'get_label' );
+		$this->assertTrue( ! empty( $res ) );
+		$this->assertTrue( is_string( $res ) );
+
+	}
+
+	/**
+	 * Test get_text()
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function test_get_text() {
+
+		$res = LLMS_Unit_Test_Util::call_method( $this->main, 'get_text' );
+		$this->assertTrue( ! empty( $res ) );
+		$this->assertTrue( is_string( $res ) );
+
+	}
+
+	/**
+	 * Test get_orders() during a cache hit
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function test_get_orders_cache_hit() {
+
+		wp_cache_set( 'recurring-payment-rescheduler', 'mock cache', 'llms_tool_data' );
+		$this->assertEquals( 'mock cache',  LLMS_Unit_Test_Util::call_method( $this->main, 'get_orders' ) );
+
+	}
+
+	/**
+	 * Test get_orders() during a cache miss
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function test_get_orders_cache_miss() {
+
+		$orders = $this->create_orders_to_handle();
+
+		// Order IDs returned.
+		$this->assertEqualSets( $orders, LLMS_Unit_Test_Util::call_method( $this->main, 'get_orders' ) );
+
+		// Cache is set.
+		$this->assertEqualSets( $orders, wp_cache_get( 'recurring-payment-rescheduler', 'llms_tool_data' ) );
+
+	}
+
+	/**
+	 * Test handle()
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function test_handle() {
+
+		$orders = $this->create_orders_to_handle();
+
+		$res = LLMS_Unit_Test_Util::call_method( $this->main, 'handle' );
+
+		// All expected orders were handled.
+		$this->assertEqualSets( $orders, $res );
+
+		// Cache erased.
+		$this->assertFalse( wp_cache_get( 'recurring-payment-rescheduler', 'llms_tool_data' ) );
+
+		foreach ( $res as $id ) {
+
+			$order = llms_get_post( $id );
+
+			// Action is rescheduled.
+			$this->assertEquals( $order->get_next_payment_due_date( 'U' ), $order->get_next_scheduled_action_time( 'llms_charge_recurring_payment' ) );
+
+		}
+
+	}
+
+	/**
+	 * Test query_orders()
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function test_query_orders() {
+
+		// No orders.
+		$this->assertEquals( array(), LLMS_Unit_Test_Util::call_method( $this->main, 'query_orders' ) );
+
+		// Should be found.
+		$to_handle = $this->create_orders_to_handle();
+
+		// This order should not be in the returned array.
+		$to_ignore = $this->create_orders_to_handle( 1, false );
+
+		$res = LLMS_Unit_Test_Util::call_method( $this->main, 'query_orders' );
+
+		$this->assertEqualSets( $to_handle, wp_list_pluck( $res, 'ID' ) );
+
+	}
+
+	/**
+	 * Test should_load()
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	protected function test_should_load() {
```

Comment:
```suggestion
	public function test_should_load() {
```

Reply:
I pushed real quick at the end of the day and must have missed the warning locally, sorry.